test/testing/test_utils.py

```diff
@@ -98,51 +97,6 @@ class ServiceY(object):
     assert get_container(runner, object) is None
 
 
-def test_reset_rabbit_vhost(rabbit_config, rabbit_manager):
-
-    vhost = rabbit_config['vhost']
-    username = rabbit_config['username']
-
-    def get_active_vhosts():
-        return [vhost_data['name'] for
-                vhost_data in rabbit_manager.get_all_vhosts()]
-
-    reset_rabbit_vhost(vhost, username, rabbit_manager)
-    assert vhost in get_active_vhosts()
-
-    rabbit_manager.delete_vhost(vhost)
-    assert vhost not in get_active_vhosts()
-
-    reset_rabbit_vhost(vhost, username, rabbit_manager)
-    assert vhost in get_active_vhosts()
-
-
-def test_reset_rabbit_vhost_errors():
```

Comment:
The `reset_rabbit_vhost` helper doesn't exist anymore. We either create and later delete a random vhost, or use whichever was specified and don't modify it.